src/components/AlertCard/styles.module.css

```diff
@@ -52,16 +59,34 @@
   display: inline-block;
   height: 70px;
   width: 8px;
-  background-color: red;
+  /* background-color: red; */
   padding-left: 5px;
 }
 
+.flexCollapse {
+  display: flex;
+}
+
+.childCollapse {
+  margin-right: 15px;
+}
+
 .low {
-  color: alertYellow;
+  width: 32px;
+  height: 25px;
+  background: url(../../../public/static/medium.svg);
```

Comment:
public isn't generated until compile time. I'd suggest putting the images into the `static` folder, and then they can be referenced as urls from the root, like `url('/medium.svg')`.

Reply:
fixed